frontend/src/bundles/common/components/transaction-modal/transaction-image.tsx

```diff
@@ -0,0 +1,50 @@
+import { faCamera } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { useRef } from 'react';
+
+import { useCallback } from '~/bundles/common/hooks/hooks';
+
+import styles from './styles.module.scss';
+
+interface Properties {
+    file?: File;
+    handleFileChange: (event: React.ChangeEvent<HTMLInputElement>) => void;
+}
+
+const TransactionImage: React.FC<Properties> = ({ file, handleFileChange }) => {
+    const inputReference = useRef<HTMLInputElement>(null);
+
+    const handleClick = useCallback((): void => {
+        if (inputReference.current) {
+            inputReference.current.click();
+        }
+    }, []);
+
+    return (
+        <button
+            type="button"
+            className={styles.imageButton}
+            onClick={handleClick}
+        >
+            {file ? (
+                <img
+                    className={styles.image}
+                    src={URL.createObjectURL(file)}
+                    alt="transaction"
+                />
+            ) : (
+                <FontAwesomeIcon icon={faCamera} />
+            )}
+            <input
```

Comment:
we have a common component

Reply:
Not resolved

Reply:
Could you use `<Input/>` component from `bundles/common/components`?